src/Builder/CommonBuilder.php

```diff
@@ -123,7 +123,12 @@ protected function createBasicTags()
 
         $tags['a']->addAttrs(array_merge(
             [$this->hrefAttr],
-            $this->createAttrs('hreflang', 'rel', 'referrerpolicy', 'target', 'type')
+            $this->createAttrs(
+                // https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a
+                'download', 'hreflang', 'media', 'ping', 'rel', 'referrerpolicy', 'target', 'type',
```

Comment:
`media` doesn't look right according to the aforementioned website.

Reply:
Awesome... while navigating on MDN browser language switched to German, I've taken `media` from that outdated data:
https://developer.mozilla.org/de/docs/Web/HTML/Element/a#attr-media